clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -38,7 +50,24 @@
                     ".cob"
                 ]
             }
-        ]
+        ],
+        "configuration": {
+            "title": "Broadcom COBOL LSP",
+            "properties": {
+                "broadcom-cobol-lsp.cpy-manager.profiles": {
```

Comment:
can the profile be a string?

Reply:
Zowe Profile is a pretty complex structure, here I use an object to store relation with program name (string) to profile name (string)